docs/style-guide.md

```diff
@@ -85,27 +115,90 @@ There are three areas which may warrant a technically public API that we don't c
 * Where we want to expose some non-public API for testing or benchmarking
 * Where we want to expose some non-public API for helper macros
 
-### No public fields :: suggested
+### Public fields vs getter/setter :: required
```

Comment:
Should you discuss the Default trait in this section as Suggested for the option bags?